vars/withOpenshiftServices.groovy

```diff
@@ -100,7 +100,7 @@ Map<String, String> getNames(List<String> services) {
 
 List<String> env(List<String> services, List<String> names) {
     List<String> out = []
-    if (services.contains('mongodb')) {
+    if (services.contains('mongodb') || services.contains('mongodb32')) {
         out.add("MONGODB_HOST=${names[services.indexOf('mongodb')]}")
```

Comment:
We're referencing mongodb here. Should this not cater for both mongodb and mongodb32?

Reply:
@pmccarthy It should, yes. I'll change it.

For anyone else reading this, the reason it still worked in my testing, even though I forgot to change it is interesting:

The `.indexOf` method on a List [comes from Java](http://docs.oracle.com/javase/8/docs/api/java/util/List.html#indexOf-java.lang.Object-)
> Returns:
    the index of the first occurrence of the specified element in this list, _**or -1 if this list does not contain the element**_

In Groovy though, one can use negative indices with the [subscript operator](http://groovy-lang.org/operators.html#subscript-operator) to count from the end of the list.

I had a list of 1, so this unintentionally worked. When changing this, I'll see if there's another way to do this that will be less scary.

Reply:
@pmccarthy I've fixed that now, would you mind taking another look?